source/content/guides/integrated-composer/01-introduction.md

```diff
@@ -43,7 +43,7 @@ You can use the [Terminus Conversion Tools Plugin](https://github.com/pantheon-s
 
 <Alert title="Note" type="info">
 
-Pantheon has a [WordPress (Composer Managed)](/guides/wordpress-composer/pre-ga/wordpress-composer-managed) upstream. You can use this upstream to create an Integrated Composer WordPress site with **Bedrock**. This upstream is currently in EA.
+Pantheon has a [WordPress (Composer Managed)](/guides/wordpress-composer/wordpress-composer-managed) upstream. You can use this upstream to create an Integrated Composer WordPress site with **Bedrock**. This upstream is currently in EA.
```

Comment:
@jazzsequence The section that this line appears in is very confusing. Does the removal of the pre-ga URL also mean we are getting rid of the documentation on https://github.com/pantheon-upstreams/wordpress-project

Reply:
We aren't getting rid of `wordpress-project` until WordPress Composer Managed is supported by build tools. Today it is not.